network-api/networkapi/wagtailpages/templates/wagtailpages/static/initiatives_page.html

```diff
@@ -7,7 +7,7 @@
 {% image page.primaryHero fill-2400x700-c100 format-jpeg as big_hero %}
 <div style="background-image: url({{ big_hero.url }}); background-repeat:no-repeat; background-position: top center; background-size: cover; padding-top: 100px;">
 
-  <div class="container p-3 p-sm-5 mx-4 mx-sm-auto mb-4 bg-white">
+  <div class="container p-3 p-sm-5 mx-sm-auto mb-4 bg-white">
```

Comment:
This one was _amazing_

Reply:
This change fixes the overflow issue but breaks the design in a different way. 😬 
 
Mockup: (notice the "cutout" effect for the intro section)
<img width="370" alt="image" src="https://user-images.githubusercontent.com/2896608/83808496-bc59cc80-a669-11ea-82b6-bf79084b03e6.png">

Review app:
<img width="370" alt="image" src="https://user-images.githubusercontent.com/2896608/83808708-1c507300-a66a-11ea-9762-8d1b255275cb.png">

Reply:
Removed the margins on the block itself, and instead turned them into padding on the containing div. That way we don't need to worry about "width + margin".